src/api/views.py

```diff
@@ -0,0 +1,55 @@
+from django.http import JsonResponse
+from django.db.models import F
+from django.views.generic.base import View
+
+from sponsors.models import Sponsor, OpenRole
+
+class SponsorAPIView(View):
```

Comment:
nit: Personally prefer moving sponsor-related API views to `sponsor` app and then imported in `api/urls.py`. We can modify it till we feel it's annoying.